iceoryx_hoofs/include/iceoryx_hoofs/cxx/algorithm.hpp

```diff
@@ -99,17 +99,19 @@ template <typename T, typename CompareType, typename Next, typename... Remainder
 constexpr bool doesContainType() noexcept;
 
 /// @brief Finalizes the recursion of doesContainValue
-/// @param[in] v type of the value to check
 /// @return always false
 template <typename T>
-inline constexpr bool doesContainValue(const T v) noexcept;
+inline constexpr bool doesContainValue(const T) noexcept;
 
 /// @brief Returns true if value of T is found in the ValueList, otherwise false
 /// @tparam T type of the value to check
 /// @tparam ValueList is a list of values to check for a specific value
 /// @param[in] value to look for in the ValueList
 /// @param[in] firstValueListEntry is the first variadic argument of ValueList
 /// @param[in] remainingValueListEntries are the remaining variadic arguments of ValueList
+/// @return true if value is contained in the ValueList, otherwise false
+/// @note be aware that value is tested for exact equality with the entries of ValueList and regular floating-point
```

Comment:
:heart: Maybe mention that comparing float exactly typically doesn't make much sense?

Reply:
In my opinion the current documentation is sufficient. @MatthiasKillat What do you think?

Reply:
Whether it makes sense or not depends on context. In generic code w do not have this context so this is fine.